<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AuthorizationForm</class>
 <widget class="QWidget" name="AuthorizationForm">
  <property name="enabled">
   <bool>false</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>191</width>
    <height>144</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AuthorizationForm</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLineEdit" name="loginEdit">
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Login</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="passwordEdit">
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>Password</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="haveAccessCode">
     <property name="text">
      <string>I have an access code</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="accessCodeEdit">
     <property name="placeholderText">
      <string>Your access code</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="signInBtn">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Sign in</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="signUpBtn">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Sign up</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
